docs/tutorial/getting-started.rst

```diff
@@ -14,20 +14,23 @@ Prerequisites
 
 * Ubuntu 22.04.
 * An Internet connection.
-* DSS relies on `MicroK8s`_, that requires as little as 540MB of memory.
-  But to accommodate workloads, we recommend a system with at least 20G
-  of disk space and 4G of memory.
+* `Snap`_.
 
 .. _set_microk8s:
 
 Setting up MicroK8s
 -------------------
 
 DSS relies on a container orchestration system, capable of exposing the host GPUs to the workloads. 
-In this tutorial, MicroK8s is used as the orchestration system.
+`MicroK8s`_ is used as the orchestration system.
 All the workloads and state managed by DSS will be running on top of MicroK8s.
 
-You can install MicroK8s using `snap`_ as follows:
+.. note::
+
+   MicroK8s requires 540MB of memory.
```

Comment:
Does microK8s have a page for minimum requirements and if yes would it make sense for us to link it here?

Reply:
Linking is always helpful. Added.